patches/net/minecraft/client/Minecraft.java.patch

```diff
@@ -76,9 +76,10 @@
          this.setOverlay(
 -            new LoadingOverlay(
 -                this, reloadinstance, p_299779_ -> Util.ifElse(p_299779_, p_299772_ -> this.rollbackResourcePacks(p_299772_, minecraft$gameloadcookie), () -> {
+-                        if (SharedConstants.IS_RUNNING_IN_IDE) {
 +            net.neoforged.fml.loading.ImmediateWindowHandler.<LoadingOverlay>loadingOverlay(
 +                () -> this, () -> reloadinstance, p_299779_ -> Util.ifElse(p_299779_, p_299772_ -> this.rollbackResourcePacks(p_299772_, minecraft$gameloadcookie), () -> {
-                         if (SharedConstants.IS_RUNNING_IN_IDE) {
++                        if (false) {
```

Comment:
Do we maybe want to make the patches be `if (false && SharedConstants.IS_RUNNING_IN_IDE) {` just to make it easier when looking at code in general in userdev what the intention of the following code was rather than potentially being something that the loader patched out due to adding a hook that replaces the logic for it?